build.gradle

```diff
@@ -0,0 +1,82 @@
+/*
+ * Licensed to Elasticsearch under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import org.elasticsearch.gradle.ElasticsearchProperties
+
+buildscript {
+  repositories {
+    mavenCentral()
+    maven {
+      name 'sonatype-snapshots'
+      url 'http://oss.sonatype.org/content/repositories/snapshots/'
+    }
+    jcenter()
+  }
+  dependencies {
+    classpath 'org.elasticsearch.gradle:build-tools:3.0.0-SNAPSHOT'
+  }
+}
+apply plugin: 'idea'
+apply plugin: 'eclipse'
+apply plugin: 'elasticsearch.esplugin'
+
+esplugin {
+  description 'The mapper attachments plugin adds the attachment type to Elasticsearch using Apache Tika.'
+  classname 'org.elasticsearch.plugin.mapper.attachments.MapperAttachmentsPlugin'
+}
+
+project.group = 'org.elasticsearch'
+project.version = ElasticsearchProperties.version
```

Comment:
Yes we did. I don't have an opinion. We can do one or the other.

If we keep old groupId, then we can release this plugin more often (not sure it makes sense though).
If we use the new groupId (ES version), we can only release when elasticsearch releases.

The easier thing to do though is to move this plugin under elasticsearch repo.